src/prepare.js

```diff
@@ -41,7 +41,7 @@ function prepareElement(element, context) {
     return Promise.resolve([null, context]);
   }
   const {type, props} = element;
-  if (typeof type === 'string') {
+  if (typeof type === 'string' || type === React.Fragment) {
```

Comment:
I feel like we should still traverse the children here

Reply:
Do you have a code example where this would be leveraged? I checked with a simple `<div>` element, and this was the path that we fall into, so I imagined it would be similar to a div case. Seems like we should handle children the same in both cases?